src/utils/parser/getSourceMappedExpression.js

```diff
@@ -0,0 +1,25 @@
+import { getScopes } from ".";
+import { replaceOriginalVariableName } from "devtools-map-bindings/src/utils";
+
+/**
+ * Gets information about original variable names from the source map
+ * and replaces all posible generated names.
+ */
+export default async function getSourceMappedExpression(
```

Comment:
Good question, I've thought a lot about testing this function and I don't think we should write a unit test for it. Let me explain:

1. we have a mochitest for the mapping in scopes
2. we have individual unit tests for all three calls. 

This function is the classic glue/contract function. If we were to test it i'd want to stub the calls so as not to repeat a test, at that point it will be a test that `replaceOriginalVariableName` is called w/ expression & generatedScopes